Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -17,7 +17,7 @@
     "demands": [],
     "version": {
         "Major": 2,
-        "Minor": 156,
+        "Minor": 157,
```

Comment:
Please leave Minor at 156 and bump Patch to 2. We're using a time-based scheme rather than Semantic Versioning here.